src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -433,3 +449,23 @@ export const MyCollectionArtworkFormFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+const tracks = {
```

Comment:
One pattern worth mentioning is that we've been moving tracking into a single hook lately, similar to 
- https://github.com/artsy/force/blob/main/src/Apps/Article/useArticleTracking.ts 
- https://github.com/artsy/force/blob/main/src/Apps/Auction/Hooks/useAuctionTracking.ts

Then its easy to mock the whole file to test, and there's one centrally located place to change, add, adjust tracking through time (versus having it littered throughout app. It's been working really nicely.